README.md

```diff
@@ -2,9 +2,10 @@
 
 ## Build Status 
 
-| 5.50 Branch   | 5.40 Branch   | 5.30 Branch   | 5.20 Branch   |
-| ------------- |:-------------:| -------------:| -------------:|
-| ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true)| ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true)| ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true)| ![Build status](https://ci.appveyor.com/api/projects/status/u84505l6syp70013?svg=true)
+OV Version | 5.60 | 5.50 | 5.40 | 5.30 |
+| ------------- |:-------------:| -------------:| -------------:| -------------:|
+SDK Version/Tag | [v1.7.0-0.11](https://github.com/HewlettPackard/terraform-provider-oneview/releases/tag/v1.7.0) |[v1.6.0](https://github.com/HewlettPackard/terraform-provider-oneview/releases/tag/v1.6.0) | [v1.5.0](https://github.com/HewlettPackard/terraform-provider-oneview/releases/tag/v1.5.0) | [v1.4.0](https://github.com/HewlettPackard/terraform-provider-oneview/releases/tag/v1.4.0) |
```

Comment:
change https://github.com/HewlettPackard/terraform-provider-oneview/releases/tag/v1.7.0 to
https://github.com/HewlettPackard/terraform-provider-oneview/releases/tag/v1.7.0-11